environment.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
     - jinja2
     - redis
     - redis-py
+    - tika=1.8.7
     - pip:
         - tld
         - selenium
```

Comment:
@chrismattmann - This is where we'd put a `pip install tika=1.8.7` command.  Is this what you intended?